Model/ShimModel.php

```diff
@@ -812,6 +812,47 @@ public function belongsTo($associated, array $options = []) {
 		$this->_setAssoc('belongsTo', $associated, $options);
 	}
 
+	/**
+	 * Setup multiple associations.
+	 *
+	 * It takes an array containing set of table names indexed by association type
+	 * as argument:
+	 *
+	 * ```
+	 * $this->Post->addAssociations([
+	 *   'belongsTo' => [
+	 *     'User' => ['className' => 'Users']
```

Comment:
I might have been too quick: Users should be User right?

Reply:
Usually yes I think. But the class name can be plural too.

Reply:
But in 2.x it usually isnt.